src/Avalonia.Base/Data/Core/Plugins/DataAnnotationsValidationPlugin.cs

```diff
@@ -24,7 +26,7 @@ public bool Match(WeakReference<object> reference, string memberName)
         }
 
         /// <inheritdoc/>
-        public IPropertyAccessor Start(WeakReference<object> reference, string name, IPropertyAccessor inner)
+        public IPropertyAccessor Start(WeakReference<object?> reference, string name, IPropertyAccessor inner)
         {
             return new Accessor(reference, name, inner);
         }
```

Comment:
Shouldn't it return null as per IPropertyAccessorPlugin? Similarly how it is in AvaloniaPropertyAccessorPlugin.
Also in AvaloniaPropertyAccessorPlugin this method returns null early, if weak reference is not active anymore.

Reply:
Possibly/probably yeah, but as I said I tried not to change things that weren't directly related to NREs in this PR.